bin/never_alone_create_review_trail.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+set -Eeu
+
+SCRIPT_NAME="never_alone_create_review_trail.sh"
+MAIN_BRANCH=""
+COMMIT_PULL_REQUEST_FLOW=""
+PROPOSED_COMMIT=""
+TRAIL_NAME=""
+
+
+function print_help
+{
+    cat <<EOF
+Use: $SCRIPT_NAME [options]
+
+Script to get commit and pull request info for all commits to main/master branch
+and report them to Kosli
+
+Options are:
+  -h               Print this help menu
+  -m <branch>      Name of main/master branch. Required
+  -f <flow>        Name of kosli flow to report commit and pull request info to. Required
+  -c <commit-sha>  Commit sha for release we are building now. Required
+  -t <trail-name>  Name of the trail that the reviews shall be reported to
```

Comment:
-t is missing "Required" at the end of its help text

Reply:
Fixed